create-toolchain

```diff
@@ -106,11 +111,13 @@ cp -a "${mac_toolchain}"/usr/include linux/iphone/include
 # ...but-but replace the mac lib/swift/clang symlink with the linux one
 rm linux/iphone/lib/swift/clang
 cp -a linux/{host,iphone}/lib/swift/clang
+# remove borked plutil in favor of plistutil
+rm -v linux/{host,iphone}/bin/plutil
 
 info "Installing Darwin tools"
 # The hierarchy lines up with ours just right so we dont need to -C
 dl "darwin-tools-${linux_version}.tar.gz" "https://github.com/kabiroberai/darwin-tools-linux/releases/download/v${DARWIN_TOOLS_VERSION}/darwin-tools-${linux_version}.tar.gz"
```

Comment:
let's update `DARWIN_TOOLS_VERSION` before merging this (since we need your changes for plistutil). I'll cut a DT release once CI passes and I have the chance to test it.